packages/picker/src/Picker.ts

```diff
@@ -433,7 +433,7 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                 aria-haspopup="true"
                 aria-labelledby="icon label applied-label"
                 id="button"
-                class="button"
+                class=${this.quiet ? 'button sideLabel' : 'button'}
```

Comment:
What about something being `quiet` also makes it have a "side label"?

Reply:
I think this is only specific to `picker`

Reply:
I talked to @jenndiaz in spectrum-css and she told me that the might rename this `side-label` class to a more generic class name. And yes, we will have to to use this class